.snippets/code/tutorials/polkadot-sdk/parachains/zero-to-hero/pallet-benchmarking/runtime/src/configs/xcm_config.rs

```diff
@@ -0,0 +1,193 @@
+use crate::{
```

Comment:
Double checking: are all of these files actually needed? 

It seems like this step of the tutorial is about benchmarking.

Reply:
If we want to compile the runtime without modifying the template src code we need it